libs/full/async_mpi/include/hpx/async_mpi/mpi_future.hpp

```diff
@@ -80,6 +80,17 @@ namespace hpx { namespace mpi { namespace experimental {
             bool error_handler_initialized_ = false;
             int rank_ = -1;
             int size_ = -1;
+            // active futures holds the size of the future vector
+            // which is always the same as the mpi request vector
+            std::atomic<std::uint32_t> active_futures_size_;
+            // active requests returns the size of the request queue
+            std::atomic<std::uint32_t> request_queue_size_;
```

Comment:
```suggestion
            std::atomic<std::uint32_t> active_futures_size_{0};
            // active requests returns the size of the request queue
            std::atomic<std::uint32_t> request_queue_size_{0};
```
and `mpi_info() = default;` (or remove it)?